econplayground/templates/main/graph_form.html

```diff
@@ -11,6 +11,7 @@
 
 {% block js %}
     <script src="{% static 'js/build/editor.js' %}"></script>
+    window.EconPlayground = { graphType: {{graph_type}}}
```

Comment:
So this variable is already defined in base.html and we don't want to overwrite it here.
https://github.com/ccnmtl/econplayground/blob/master/econplayground/templates/base.html#L175

We can add a new variable to this object, and we'll want to check that it actually exists:
```
        var EconPlayground = {
            // All the existing data here

            {% if graph_type is not None %}
                graphType: {{graph_type}},
            {% endif %}
        };
```

Note that you'll also need to add commas to the `isStaff` variable declarations above in order for this to parse.

Reply:
Or, we can conditionally declare this object within this template, and just add this new variable to it within this template - following this pattern: https://github.com/ccnmtl/econplayground/blob/master/econplayground/templates/main/graph_detail.html#L18

Maybe that is a bit cleaner as this variable is only present for this specific view. But functionally, it doesn't matter so much either way.

Reply:
I went for the implementation that wouldn't affect the base as this variable is only necessary in graph creation.